analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt

```diff
@@ -36,21 +41,30 @@ import kotlinx.coroutines.launch
  * Main screen layout for the sample app
  */
 @Composable
-fun MainScreen(sampleName: String, application: Application) {
+fun MainScreen(sampleName: String) {
     // coroutineScope that will be cancelled when this call leaves the composition
     val sampleCoroutineScope = rememberCoroutineScope()
+    // get the application property that will be used to construct MapViewModel
+    val sampleApplication = LocalContext.current.applicationContext as Application
     // create a ViewModel to handle MapView interactions
-    val mapViewModel = remember { MapViewModel(application, sampleCoroutineScope) }
+    val mapViewModel = remember { MapViewModel(sampleApplication, sampleCoroutineScope) }
+    // create a Viewpoint
+    val viewpoint = Viewpoint(
+        center = Point(-13671170.0, 5693633.0, SpatialReference(wkid = 3857)),
+        scale = 1e5
+    )
 
     Scaffold(
         topBar = { SampleTopAppBar(title = sampleName) },
         content = {
             // sample app content layout
             Column(modifier = Modifier.fillMaxSize().padding(it)) {
-                // composable function that wraps the MapView
-                ComposeMapView(
-                    modifier = Modifier.fillMaxSize().weight(1f),
-                    mapViewModel = mapViewModel
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISMap = mapViewModel.map,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = viewpoint)
```

Comment:
could we rename the `viewpoint` variable to something else for code clarity or we could even directly substitute the Viewpoint() here without having to declare it above.
We did the same in previous samples.

Reply:
Thanks for pointing this out. I will directly substitute the Viewpoint() here to simply the code.